gitbutler-ui/src/lib/components/BranchCard.svelte

```diff
@@ -112,7 +112,7 @@
 		generateBranchName();
 	}
 
-	onMount(() => {
+	onMount(async () => {
```

Comment:
Since there is no async calls in this onMount we don't need to pass it an async arrow function.

Reply:
Thanks for spotting that. That is just something I missed when committing